### PR TITLE
Fix #189  (Dashboard Widgets errors new admin)

### LINKED ISSUE #189: DashBoard Widget errors new admin
When I make a new Admin user and then logout and try to login as that new user I get the error:
[26-Sep-2014 10:33:59 America/New_York] PHP Catchable fatal error:  Argument 1 passed to ZenCart\Admin\DashboardWidget\WidgetManager::loadWidgetClasses() must be an array, null given, called in /home/username/public_html/subdirectory/admin/includes/classes/actions/admin/class.zcActionAdminIndex.php on line 37 and defined in /home/username/public_html/subdirectory/admin/includes/library/zencart/DashboardWidget/src/WidgetManager.php on line 106

### PR DESCRIPTION
**Pull request by scottcwilson**
Fixes #189 (Dashboard Widgets errors new admin). 
This issue was reported by @Ajeh.

### CODE REVIEW

**Comment by ajeh** [comment]
:+1:

File: admin/includes/template/templates/tplIndex.php [hunk 1]
```diff
@@ -35,6 +35,7 @@
 <script>
 var intervalTimers = new Array();
 $(function() {
+  <?php if (isset($tplVars['widgetList']) && (sizeof($tplVars['widgetList'] == 0))) { ?>
```
Comment (drbyte) [inline admin/includes/template/templates/tplIndex.php]: Are the extra parentheses necessary around the `(sizeof())`

Comment (drbyte) [inline admin/includes/template/templates/tplIndex.php]: And, shouldn't the closing paren for sizeof be before the `== 0`?

I suggest:
```php
<?php if (isset($tplVars['widgetList']) && sizeof($tplVars['widgetList']) == 0) { ?> 
```

Comment (scottcwilson) [inline admin/includes/template/templates/tplIndex.php]: Argh you are correct.  I will fix.  Thanks for catching that.

**Comment by drbyte** [comment]
:+1:

**Comment by zcwilt** [comment]
:+1:

**Pull request merged**